fluent-plugin-kubernetes_metadata_filter.gemspec

```diff
@@ -17,7 +17,7 @@ Gem::Specification.new do |gem|
 
   gem.required_ruby_version = '>= 2.6.0'
```

Comment:
Technically this could/should be 2.7.0 to match the upstream fluentd requirement more explicitly, but it'll resolve that way anyway.

Reply:
Thanks, it should be so.